plugins/okta/komand_okta/tasks/monitor_logs/task.py

```diff
@@ -72,3 +72,14 @@ def get_next_page_link(headers: dict) -> str:
             matched_link = re.match("<(.*?)>", link) if 'rel="next"' in link else None
             next_link = matched_link.group(1) if matched_link else None
         return next_link
+
+    def get_last_collection_timestamp(self, now, new_logs):
```

Comment:
this could be updated to have implicit params and returns like the method above 


`def get_last_collection_timestamp(self, now: datetime, new_logs: list) -> str:`

Reply:
Makes sense, not used to including the type hints yet 🤦 updated now!